tasks/dataset-merge/task.yaml

```diff
@@ -0,0 +1,21 @@
+id: merge-dataset
+name: merge-dataset
+description: This task is used to merge user-defined numbers of coretex datasets of the same
+  type into a single coretex dataset
+is_active: true
+project_type: 8
+param_groups:
+  - name: inputs
+    params:
+      - name: datasetsList
```

Comment:
`datasets` is enough, there is no need for `List` suffix, since `datasets` already indicates that it is a list.